jdbc/src/test/java/org/jclouds/jdbc/BaseJdbcBlobStoreTest.java

```diff
@@ -0,0 +1,746 @@
+/*
```

Comment:
What is this test?  Why don't you get the same functionality from running `mvn integration-test jdbcblobstore`?